dashboard/app/models/datablock_storage_table.rb

```diff
@@ -11,6 +11,8 @@
 #
 class DatablockStorageTable < ApplicationRecord
   self.primary_keys = :channel_id, :table_name
+  has_many :records, class_name: 'DatablockStorageRecord', foreign_key: [:channel_id, :table_name]
+  after_initialize -> {self.columns ||= ['id']}, if: :new_record?
```

Comment:
This sets table.columns = ['id] when you create a new table, we used to do this in the controller create_table before, but it would overwrite the existing columns if. you tried to create_table with an existing column name